hack/build-clusteradm-image.sh

```diff
@@ -71,13 +72,8 @@ clusteradm_version=${clusteradm_version#v}
 
 echo "Using clusteradm v${clusteradm_version}."
 
-git clone -b "v$clusteradm_version" --depth 1 https://github.com/open-cluster-management-io/clusteradm.git "$clusteradm_folder"
-
 cd "$clusteradm_folder"
+git clone -b "v$clusteradm_version" --depth 1 https://github.com/open-cluster-management-io/clusteradm.git .
```

Comment:
Ugh, sorry I suggested this. It was more informative the other way. Now the user of the script is not told the name of the build directory, instead is told "Cloning into '.'..." and that's confusing. Can be fixed now or in follow-on.